style: updates some components with semantic tokens

anechol opened the pull request:
# Description
Updates some components with updated semantic tokens to better align with designs.

Fixes #(issue)

## Type of change

- [x] style (adds, remove, or updates component styles)

# How Has This Been Tested?

- [ ] unit tests
- [ ] e2e tests
- [ ] accessibility tests
- [x] tested manually
- [ ] other:

**Test Configuration**:

- Pine versions:
- OS:
- Browsers:
- Screen readers:
- Misc:

# Checklist:

If not applicable, leave options unchecked.

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing tests pass locally with my changes
- [ ] Design has QA'ed and approved this PR

Commit:
style: updates some components with semantic tokens

ju-Skinner commented on libs/core/src/components/pds-chip/pds-chip.scss:
```diff
@@ -2,22 +2,22 @@
 
   --color-background-accent-dot: var(--pine-color-purple-600);
-  --color-background-accent-hover: var(--pine-color-purple-200);
+  --color-background-accent-hover: var(--pine-color-purple-300);
```
we should be using `--pine-chip-color-*-hover`. This will need to be added to `libs/core/src/global/styles/tokens/components/chip.json`

ju-Skinner commented on libs/core/src/components/pds-chip/pds-chip.scss:
```diff
@@ -34,12 +34,12 @@
-  neutral: var(--pine-color-neutral),
-  success: var(--pine-color-success),
-  warning: var(--pine-color-warning),
+  accent: var(--pine-color-purple-100),
```
this should use `--pine-chip-color-accent`

ju-Skinner commented on libs/core/src/components/pds-input/pds-input.scss:
```diff
@@ -78,8 +78,7 @@ input {
 
-
-    background-color: var(--pine-color-danger);
+    background-color: var(--pine-color-red-050);
```
may need to introduce `pine-input-background-color`

ju-Skinner committed:
style(chip): update chip to use component semantic tokens

ju-Skinner committed:
style(input): create new component specific semantic tokens

created input.json for tokenstudio

updated both the themes and metadata.json files

ju-Skinner reviewed (approved):
I added two commits, one for `chip` and the other for `input`. Give them a look at them, if they are good, please feel free to merge.
